contracts/src/v0.8/automation/dev/test/BaseTest.t.sol

```diff
@@ -63,7 +63,9 @@ contract BaseTest is Test {
   uint256 internal constant SIGNING_KEY3 = 0x80f14b11da94ae7f29d9a7713ea13dc838e31960a5c0f2baf45ed458947b730a;
   address[] internal SIGNERS = new address[](4);
   address[] internal TRANSMITTERS = new address[](4);
+  address[] internal NEW_TRANSMITTERS = new address[](4);
   address[] internal PAYEES = new address[](4);
+  address[] internal NEW_PAYEES = new address[](4);
```

Comment:
nitpick, but I think these might make more sense to be defined in the test contract? My thoughts on Base.t.sol is that it would hold common entities and code that _most_ tests rely on. But if it's only used in 1 or two places, maybe better to define it there?